test/sql/function/timestamp/test_icu_datepart.test

```diff
@@ -214,18 +214,42 @@ NULL	NULL
 query II
 SELECT isodow(ts), isodow(ts::TIMESTAMP) FROM timestamps;
 ----
-4	5
-3	2
-5	4
-2	1
-6	5
-2	1
-2	1
-6	5
+3	5
```

Comment:
I think we need to rebase this off my `hawkfish:gregorian-delta` branch because these values should match now.

Reply:
ok -- rebase done. think it will look correct once your PR merges.